.github/workflows/node.js.yml

```diff
@@ -1,29 +1,30 @@
+
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
 name: Node.js CI
 
 on:
   push:
-    branches: [main]
+    branches: [ $default-branch ]
   pull_request:
-    branches: [main]
+    branches: [ $default-branch ]
```

Comment:
Is there any significance of using `$default-branch` instead of `main`?

Reply:
Just best practices

Reply:
Got it.

Reply:
Let's merge this in and test with a small PR.